server/model/EntriesModel.js

```diff
@@ -1,47 +1,53 @@
 import moment from 'moment';
 import uuidV4 from 'uuid/v4';
 
+const entries = [];
 class EntriesModel {
-    constructor(data) {
-        this.entries = [];
-    }
-
     add(data) {
-        const entry = {
+        let entry = {
             id: uuidV4(),
             title: data.title,
             story: data.story,
             createdDate: moment(),
             modifiedDate: moment(),
-        };
-        this.entries.push(entry); 
+        };  
+        if (data.title === undefined || data.story === undefined) {
+            entry = {};
+            entries.push();
+        } else if (data.title && data.story) {
+            entries.push(entry); 
+        }                              
         return entry; 
     }
 
     findAll() {
-        return this.entries;
+        return entries;
     }
 
     findOne(id) {
-        for (let i = 0; i < this.entries.length; i += 1) {
-            if (this.entries[i].id === id) {
-                return this.entries[i];
-            }
-        }
-        return {};
+        for (const entry of entries) {
```

Comment:
entries is an array, use es6 array iterators an example: `entries.forEach()` ??

Reply:
ok, thanks will work on it